### PR TITLE
fix/Fix defects found by am commands test

### PR DESCRIPTION
**Pull request by okozlovlux**
- fixed defect in UT for OnDeviceStateChangedNotification during removing device

Related to APPLINK-27526, APPLINK-27146

### CODE REVIEW

**Comment by okozlovlux** [comment]
@LevchenkoS @anosach-luxoft @vlantonov @pvvasilev @AByzhynar @wolfylambova22 please review

File: src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc [hunk 1]
```diff
@@ -896,7 +896,7 @@ TEST_F(HMICommandsNotificationsTest,
   utils::SharedPtr<Command> command =
       CreateCommand<OnDeviceStateChangedNotification>(message);
 
-  std::string device_id = "device_id";
+  std::string device_id = "AA15F2204D6B";
```
Comment (vlantonov) [inline src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc]: @okozlovlux Could `"AA15F2204D6B"` be exposed as constant? Also could `device_id` be a `const std::string`?

Comment (okozlovlux) [inline src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc]: @vlantonov , No. It couldn't, becase in next line `EXPECT_CALL(policy_interface_, RemoveDevice(_)).WillOnce(GetArg(&device_id));` the value of argument of `RemoveDevice(_)` is storing in that `device_id`

Comment (vlantonov) [inline src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc]: @okozlovlux OK. Could "AA15F2204D6B" be exposed as constant?

Comment (okozlovlux) [inline src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc]: @vlantonov , I changed it in 7293c52. It couldn't be const.

**Comment by vlantonov** [comment]
@okozlovlux Reviewed

**Comment by pvvasilev** [comment]
@okozlovlux Reviewed.

**Comment by wolfylambova22** [comment]
@okozlovlux Reviewed.

**Comment by LevchenkoS** [comment]
@okozlovlux Reviewed.

**Comment by okozlovlux** [comment]
@anosach-luxoft @AByzhynar please review

**Comment by AByzhynar** [comment]
Reviewed

File: src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc [hunk 2]
```diff
@@ -851,7 +851,7 @@ TEST_F(HMICommandsNotificationsTest,
 
 TEST_F(HMICommandsNotificationsTest,
        OnDeviceStateChangedNotificationDeviceStateUnpaired) {
-  std::string device_id = "device_id";
+  const std::string device_id = "AA15F2204D6B";
```
Comment (anosach-luxoft) [inline src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc]: @okozlovlux why `AA15F2204D6B`?

Comment (okozlovlux) [inline src/components/application_manager/test/commands/hmi/hmi_notifications/hmi_notifications_test.cc]: @anosach-luxoft , it is just a random MAC adress for test. It must contain 12 symbols. The previous contained 9 symbols and that is why test fails.

**Comment by anosach-luxoft** [comment]
Reviewed

**Comment by wolfylambova22** [comment]
Reviewed.

**Comment by AByzhynar** [comment]
@okozlovlux Please squash commits

**Comment by okozlovlux** [comment]
@AByzhynar , done 1896ef4

**Pull request merged**